src/components/src/ProductDisplayItem/productdisplayitem.main.tsx

```diff
@@ -736,7 +735,7 @@ class ProductDisplayItemMain extends Component<ProductDisplayItemMainProps, Prod
             className="ep-btn primary wide btn-itemdetail-addtocart dropdown-toggle"
             data-toggle="dropdown"
             disabled={!availability || !productData._addtocartform}
-            id="product_display_item_add_to_cart_button"
+            id="product_display_item_add_to_cart_button-dropdown"
```

Comment:
Adding to wish lists is disabled if Req Lists are available. If the resource is displayed, we shouldn't block Wish Lists, especially since we display them in the account menu. We can confirm with Randy if needed, but we shouldn't manually hide the action:

```
{requisitionListData && Config.b2b.req_list ? (
                          <SelectRequisitionListButton />
                        ) : (
                          <div className="form-content form-content-submit col-sm-offset-4">
                            <button
                              onClick={this.addToWishList}
                              className="ep-btn wide btn-itemdetail-addtowishlist"
                              disabled={!availability || !productData._addtowishlistform}
                              id="product_display_item_add_to_wish_list_button"
                              type="submit"
                            >
                              {intl.get('add-to-wish-list')}
                            </button>
                          </div>
                        )}
```

Reply:
@shaunmaharaj thank you

Reply:
@shaunmaharaj We moved "wishlist" button to top due to mockups
<img width="1069" alt="Untitled" src="https://user-images.githubusercontent.com/34774987/74243349-44e1e500-4ce8-11ea-8880-da5b2b047f53.png">

Reply:
Thanks @rostyk-kanafotskyy, I see it now